### ISSUE TITLE
Characters move crazy fast in main menu with VSync disabled and high FPS

### ISSUE THREAD

**Issue by tomasz1986** [bolrog/d2dx#92]
This is with `novsync=true` and the FPS limited to 200. The character animations in the main menu seem to loop crazily fast, depending on the current FPS.

https://youtu.be/rflTnCppFD8

**Comment by bolrog** [comment]
Yup, that's a known issue. The characters in the main menu advance one animation frame per frame rendered, unlike the ones in-game which are tied to the fixed 25fps tick.

**Comment by fapmilk** [comment]
> Yup, that's a known issue. The characters in the main menu advance one animation frame per frame rendered, unlike the ones in-game which are tied to the fixed 25fps tick.

Any chance you can add an option to not apply the high fps / motion prediction to the main menu? Or perhaps an option to set a custom framerate cap in the main menu.

**Comment by bolrog** [comment]
That's a good idea. Ideally I would like to find where the frame number is incremented and make it move at the proper rate instead, but I will keep your suggestions in mind as a backup solution.

I'll probably add a frame cap eventually too. Just need to figure out the best way to do that.
